src/ts/greedy-nav/GreedyNav.ts

```diff
@@ -0,0 +1,1062 @@
+                        addClass(_this, 'is-open');
+                        updateLabel(
+                            _this,
+                            navDropdownLabelActive,
+                            this.navDropdownToggleSelector,
+                            this.settings.navDropdownLabelActive
+                        );
+
+                        /**
+                         * Toggle aria hidden for accessibility
+                         */
+                        navDropdown.setAttribute('aria-hidden', 'false');
+                        navDropdown.blur();
+                    }
+                }
+            });
+        }
+
+        if (navDropdownToggle) {
+            navDropdownToggle.addEventListener(
+                blurEventName,
+                (e: FocusEvent) => {
+                    if (this.toggleWrapper === null) {
+                        return;
+                    }
+
+                    const navDropdownLink = document.querySelector<HTMLElement>(
+                        `${this.navDropdownSelector} li:last-child a`
+                    );
+
+                    const navDropdown = document.querySelector<HTMLElement>(
+                        this.navDropdownSelector
+                    );
+
+                    if (
+                        !parent(
+                            <HTMLElement>e.relatedTarget,
+                            this.toggleWrapper
+                        ) &&
+                        navDropdownLink &&
+                        navDropdown
+                    ) {
+                        navDropdownLink.removeEventListener(
+                            blurEventName,
+                            lastItemCloseHandler
+                        );
+
+                        removeClass(navDropdown, 'show');
+                        if (
+                            e.currentTarget !== null &&
+                            e.currentTarget instanceof HTMLElement
+                        ) {
+                            removeClass(e.currentTarget, 'is-open');
+                        }
+                        removeClass(_this, 'is-open');
+
+                        updateLabel(
+                            _this,
+                            navDropdownLabel,
+                            this.navDropdownToggleSelector,
+                            this.settings.navDropdownLabelActive
+                        );
+
+                        /**
+                         * Toggle aria hidden for accessibility
+                         */
+                        navDropdown.setAttribute('aria-hidden', 'false');
+                    } else if (navDropdownLink) {
+                        navDropdownLink.addEventListener(
+                            blurEventName,
+                            lastItemCloseHandler
+                        );
+                    }
+                }
+            );
+        }
+
+        /*
+         * Close menu when last item is selected
```

Comment:
Seems to be a misplaced comment block...

Reply:
Thanks - I'll get it cleaned up.